camera/buildDetector.py

```diff
@@ -3,8 +3,8 @@
 import numpy as np
 
 # This is the readout noise and gain of our amps: 
-readout = [[22.3,23.1],[23.0,22.1],[23.0,24.4]]
-gain_all = [[0.53,0.54],[0.52,0.52],[0.59,0.59]]
+readout = [[12,12,12,12]]
+gain_all = [[0.53,0.54,0.54,0.54],[0.52,0.52],[0.59,0.59]]  ##not sure the value
```

Comment:
I presume you can get rid of the 0.52s and the 0.59s, since you only have one ccd 
```suggestion
gain_all = [[0.53,0.54,0.54,0.54]]  ##not sure the value
```